### ISSUE TITLE
Search results page — Accessibility updates

### ISSUE THREAD

**Issue by sshetenhelm** [yalelibrary/YUL-DC#3197]
**Issues**
As per #3068 

Low contrast in body text (form and field labels, metadata, etc.).

Facet dropdowns: Focusable elements may be missing focus styling - facet labels are missing focus. Screen reader users will not be able to tell what the facet is and will be unable to expand it if there is no focus on the facet label itself (links inside the facet receive focus by default).

Tracy also noted that thumbnails should have alt text, but currently there’s not a fix as the search results are generated dynamically. Just noting for future reference. 

**Acceptance**

- [x] Change font color for form and field labels, metadata, etc. to #4a4a4a (#4a4a4a is a recommended gray from the Yale identity websites guidelines - https://yaleidentity.yale.edu/guidelines/websites) 

- [x] Add focus styling to facet labels. 

CodePen example for button focus styling: [CodePen example for button focus styling: https://codepen.io/pen](https://github.com/orgs/yalelibrary/projects/1/views/url) / Reference: [https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Keyboard?utm_source=devtools&utm_medium=a11y-panel-checks-keyboard#Focusable_element_must_have_focus_styling](https://github.com/orgs/yalelibrary/projects/1/views/url)

**Comment by jpengst** [comment]
PR: https://github.com/yalelibrary/yul-dc-blacklight/pull/1176
